src/api/publicationApi.ts

```diff
@@ -14,6 +14,7 @@ export enum PublicationsApiPaths {
   INSERT_RESOURCE = '/publications/insert-resource',
   UPDATE_RESOURCE = '/publications/update-resource',
   FETCH_RESOURCE = '/publications/fetch-resource',
+  FETCH_MY_RESOURCES = '/publications/fetch-my-resources',
```

Comment:
synes det er litt rart at vi kaller publikasjoner endepunktene for `resources`, når vi har brukt termen `publications` over alt på det som rendres

Reply:
vi får ta en prat med backend